content/docs/tutorials/converting/discoverable.md

```diff
@@ -81,7 +85,7 @@ For our news article, **add** the following structured data to the bottom of the
 
 
 {% call callout('Note', type='note') %}
-The content should always be the same. For news articles, specify the “NewsArticle” type. The headline should match your article’s title. The image object refers to the hero image of the article.
+The context should always be the same. For news articles, specify the “NewsArticle” type. The headline should match your article’s title. The image object refers to the hero image of the article.
```

Comment:
I think "content" should remain.  It's the content of the structured data that remains the same, but the type needs to change.

Reply:
+1.

Reply:
Got it, though I'm not sure I understand what we're trying to say by "the content should always be the same."  Don't we want developers to change the other fields as well (like the datePublished, the author, etc.)?